tox.ini

```diff
@@ -31,6 +31,7 @@ changedir=.
 whitelist_externals =
     sh
 deps =
+    ply
```

Comment:
Shouldn't be needed, it's a dependency of prophy and it installs via tox. In other words, it's not a test environment dependency.